### DISCUSSION TITLE
audit: telemetry

### DISCUSSION THREAD

**Discussion by user82622** [open-webui/open-webui#807] [General]
Many users assume this is a private alternative to the big cloud LLM's, the Ollama open webui seems to use lots of telemetry according to my network scans. There is not even an opt out option or any mentioning of this in the whole git or on the website. This is extremely misleading and dangerous cause the Open Webui is literally advertised as private!

**Comment by justinh-rahb** [comment]
Please share your network scans, we have a history of addressing these issues quickly; **we do not wish to have unknown telemetry anymore than you do**. These can be snuck in by any of our many hundreds of upstream dependency packages, at any time.

- #462 
- #687

**Reply by qhaas** [reply]
> we do not wish to have unknown telemetry anymore than you do. These can be snuck in by any of our many hundreds of upstream dependency packages, at any time.

As a user, I greatly appreciate the open-webui team disabling the 'telemetry' of said dependencies.  It looks like a rabbit hole and in addition to the ones getting added upstream, the existing ones might even change how they are disabled over time.  I just looked at the dockerfile, sorting this out took some time:
https://github.com/open-webui/open-webui/blob/main/Dockerfile#L65
https://github.com/open-webui/open-webui/blob/main/Dockerfile#L66
https://github.com/open-webui/open-webui/blob/main/Dockerfile#L67
https://github.com/open-webui/open-webui/blob/main/Dockerfile#L103

Thanks!

**Comment by justinh-rahb** [comment]
From a cursory examination, I'm not finding anything fishy as of now.

**Comment by user82622** [comment]
yeah, i will have to do it properly tomorrow or so this looked fine to me too but i just did it with netstat inside the container. I cant remember what action in the UI i took that was spiking connections, let me further test and send you logs as soon as i have results again.

**Reply by spergware** [reply]
Any news on that?

**Comment by tjbck** [comment]
@user82622 much appreciated!

**Reply by commit4ever** [reply]
Hi 
is there an update on the opt in ?

Thanks.

**Reply by tjbck** [reply]
@aiseei we don't collect any data, you don't have to opt out of anything :)
